service/tracker/execution.go

```diff
@@ -167,6 +167,7 @@ func (e *Execution) processNext() error {
 	// Check if we already processed a block with this ID recently. This should
 	// be idempotent, but we should be aware if something like this happens.
 	blockID := record.Block.Header.ID()
+
```

Comment:
For me

```go
	_, ok := e.records[blockID]
	if ok {
		return fmt.Errorf("duplicate execution record (block: %x)", blockID)
	}
```

Can't be cuddled with any other statement, since it's a variable definition followed by an if statement that uses said variable.

I agree that here the definition of `blockID := record.Block.Header.ID()` is integral to this block, but I still wouldn't cuddle them normally. I guess once more, we have different habits and a different way to read Go code :p

Reply:
It's the same as:

```go
blockID := header.ID()
height, err := index.HeigthByBlockID(blockID)
if err != nil { ... }
```

Makes no sense to put it on its own line.